src/transformers/models/umt5/__init__.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2020 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import TYPE_CHECKING
+
+from ...utils import (
+    OptionalDependencyNotAvailable,
+    _LazyModule,
+    is_flax_available,
+    is_sentencepiece_available,
+    is_tf_available,
+    is_tokenizers_available,
+    is_torch_available,
+)
+
+
+if is_sentencepiece_available():
+    from ..t5.tokenization_t5 import T5Tokenizer
+else:
+    from ...utils.dummy_sentencepiece_objects import T5Tokenizer
+
+UMT5Tokenizer = T5Tokenizer
```

Comment:
We don't do things like this anymore 😅 Now we just recommend using AutoTokenizer if the tokenizer is form another model! Take a look at what was done for `OPT` (just saying took me a long time to find where `UMT5Tokennizer` came from

Reply:
Thanks for your feedback, but I could not find the logic behind it in OPT. I could not find any thing related to tokenizer on the init file.
Could you explain the necessary changes?

Reply:
Yep basically you just don't create a `UMT5Tokenizer`, and only add the mapping in `tokenization_auto` with `            ("umt5", ("T5Tokenizer" if is_sentencepiece_available() else None, "T5TokenizerFast" if is_tokenizers_available() else None)),`

Reply:
Thanks for the information, I had also to change other files but now it is using T5Tokenizer as you recommended.